dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -307,8 +309,234 @@ public void findOneTestWrongUUID() throws Exception {
         Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
 
         getClient().perform(get("/api/core/items/" + UUID.randomUUID()))
-                   .andExpect(status().isNotFound());
+                   .andExpect(status().isNotFound())
         ;
+
+    }
+
+    @Test
+    public void widthdrawPatchTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and one collection.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1)
+                                           .withRemovePermission(true)
+                                           .withName("Collection 1").build();
+
+        //2. One public item
+        Item item = ItemBuilder.createItem(context, col1)
+                               .withTitle("Public item 1")
+                               .withIssueDate("2017-10-17")
+                               .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                               .withSubject("ExtraEntry")
+                               .withWritePermission(true)
+                               .build();
+
+        // A token must be provided for withdraw operation. The person
+        // is used in the provenance note.
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        // withdraw item
+        getClient(token).perform(patch("/api/core/items/" + item.getID())
+            .header(AUTHORIZATION_HEADER, AUTHORIZATION_TYPE + " " + token)
+            .content("[{\"op\":\"replace\",\"path\":\"/withdrawn\", \"value\": true}]")
```

Comment:
it is not required but it could be easier to instantiate a Patch object and serialize it using a Jackson ObjectMapper.
The code will be more readable and consistent as I guess it will be hard to write more complex test by hand maybe involving multiple operations etc.